src/revpi-factory-reset

```diff
@@ -8,14 +8,24 @@
 # reset dtoverlay, password of user pi, machine-id, hostname, FQDN and
 # MAC address of a RevPi to factory defaults
 
+. /usr/sbin/revpi-functions
```

Comment:
Only the program/script which needs superuser permission should be put in /usr/sbin/. For the script revpi-funtions, only normal user permission is needed, it can be put in /usr/bin/ or /usr/lib/ ...

Reply:
Ok, if I want to pread those files all over the system, I definitively want not to have it in $PATH anyways. So `/usr/lib` or `/usr/share` its going to be. Perhaps I find some other stuff I can put it next to.

Reply:
Changed to `/usr/share/revpi`